content/300-guides/050-database/870-using-prisma-with-mongodb.mdx

```diff
@@ -38,7 +38,7 @@ MongoDB's document-based structure and flexible schemas means that using Prisma
 
 - **Introspection and Prisma relations**: When you introspect an existing MongoDB database, you will get a schema with no relations and will need to add the missing relations in manually. For more information, see [How to add in missing relations after Introspection](#how-to-add-in-missing-relations-after-introspection).
 
-- **Filtering for `null` and undefined fields**: MongoDB makes a distinction between setting a field to `null` and not setting it at all, which is not present in relational databases. Prisma currently does not express this distinction, which means that you need to be careful when filtering for `null` and undefined fields. For more information, see [How to filter for `null` and undefined fields](#how-to-filter-for-null-and-undefined-fields).
```

Comment:
Doesn't each of these list items need a link down to where it is explained in more detail?

Reply:
Sorry, yes, brain fart. Put it back in.